components/root/root.tsx

```diff
@@ -358,6 +360,27 @@ export default class Root extends React.PureComponent<Props, State> {
         GlobalActions.redirectUserToDefaultTeam();
     }
 
+    trackUTMCampain() {
+        const qs = new URLSearchParams(window.location.search);
+        if (qs.has('utm_source')) {
```

Comment:
Will `utm_source` always be present for all type of campaigns @emdecr ?

Reply:
Looping in Kevin (@nat-gunner). I'm pretty sure that's a standard/default param we add, but confirming for this `if` check.

Reply:
I did consider that the three would always come together so testing once was enough, but if it is not the case I can just make sure that at least one is not empty before triggering the call.

Reply:
they _should_ always be there, but there's definitely a chance that one of the three could be missing at some point. 
probably a good idea to check for at least one non-empty param like you suggested, Julien.

Reply:
Alright, I made the change. Thanks for the input.